apollo-api/build.gradle.kts

```diff
@@ -4,6 +4,21 @@ plugins {
 }
 
 kotlin {
+  data class IosTarget(val name: String, val preset: String, val id: String)
```

Comment:
Totally optional but is it worth making this `iOSTarget` so it's easier to parse visually? I think I'm fine with `iosTargets` below but the uppercase here threw me off